app/src/styles/App.scss

```diff
@@ -793,6 +793,10 @@ $hunit: '%';
     border: none;
 }
 
+.govuk-phase-banner__content__tag {
```

Comment:
With reference to the above suggestion when using the `nhsuk-tag` we match he NHS style so this can be removed as i doesn't apply any text transformation so this can be removed. That being said `nhsuk-tag` doesn't have the `margin-right: 10px` so we could add that to the tag or add `margin-left: 10px` to the `govuk-phase-banner__text`.